2016-04-01 14:15:38 [7318] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  post_fork: <function post_fork at 0x7f33ef3b6cf8>
  pythonpath: None
  syslog: False
  syslog_facility: user
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 3
  cert_reqs: 0
  suppress_ragged_eofs: True
  when_ready: <function when_ready at 0x7f33ef3b6a28>
  pre_fork: <function pre_fork at 0x7f33ef3b6b90>
  preload_app: False
  workers: 1
  keepalive: 2
  accesslog: out
  pidfile: None
  worker_tmp_dir: None
  group: 1002
  graceful_timeout: 30
  spew: False
  proc_name: None
  umask: 0
  on_reload: <function on_reload at 0x7f33ef3b68c0>
  post_worker_init: <function post_worker_init at 0x7f33ef3b6e60>
  pre_exec: <function pre_exec at 0x7f33ef33d320>
  limit_request_fields: 100
  on_exit: <function on_exit at 0x7f33ef33d9b0>
  config: None
  logconfig: None
  check_config: False
  do_handshake_on_connect: False
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x7f33ef33d488>
  post_request: <function post_request at 0x7f33ef33d578>
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x7f33ef33d050>
  ca_certs: None
  threads: 1
  max_requests: 0
  chdir: /home/box/web
  daemon: False
  user: 1002
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  worker_exit: <function worker_exit at 0x7f33ef33d6e0>
  paste: None
  default_proc_name: web:app
  errorlog: out
  logger_class: gunicorn.glogging.Logger
  syslog_addr: udp://localhost:514
  syslog_prefix: None
  ciphers: TLSv1
  worker_abort: <function worker_abort at 0x7f33ef33d1b8>
  loglevel: debug
  bind: ['192.168.101.128:8080']
  raw_env: []
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x7f33ef33d848>
  timeout: 30
  on_starting: <function on_starting at 0x7f33ef3b6758>
  debug: False
  django_settings: None
  tmp_upload_dir: None
  keyfile: None
  backlog: 2048
2016-04-01 14:15:38 [7318] [INFO] Starting gunicorn 19.0.0
2016-04-01 14:15:38 [7318] [DEBUG] Arbiter booted
2016-04-01 14:15:38 [7318] [INFO] Listening at: http://192.168.101.128:8080 (7318)
2016-04-01 14:15:38 [7318] [INFO] Using worker: sync
2016-04-01 14:15:38 [7323] [INFO] Booting worker with pid: 7323
2016-04-01 14:15:38 [7323] [ERROR] Exception in worker process:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gunicorn/arbiter.py", line 502, in spawn_worker
    worker.init_process()
  File "/usr/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python2.7/dist-packages/gunicorn/util.py", line 365, in import_app
    __import__(module)
ImportError: No module named web
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gunicorn/arbiter.py", line 502, in spawn_worker
    worker.init_process()
  File "/usr/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python2.7/dist-packages/gunicorn/util.py", line 365, in import_app
    __import__(module)
ImportError: No module named web
2016-04-01 14:15:38 [7323] [INFO] Worker exiting (pid: 7323)
2016-04-01 14:16:13 [7334] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  post_fork: <function post_fork at 0x7fad4d46dcf8>
  pythonpath: None
  syslog: False
  syslog_facility: user
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 3
  cert_reqs: 0
  suppress_ragged_eofs: True
  when_ready: <function when_ready at 0x7fad4d46da28>
  pre_fork: <function pre_fork at 0x7fad4d46db90>
  preload_app: False
  workers: 1
  keepalive: 2
  accesslog: out
  pidfile: None
  worker_tmp_dir: None
  group: 1002
  graceful_timeout: 30
  spew: False
  proc_name: None
  umask: 0
  on_reload: <function on_reload at 0x7fad4d46d8c0>
  post_worker_init: <function post_worker_init at 0x7fad4d46de60>
  pre_exec: <function pre_exec at 0x7fad4d3f4320>
  limit_request_fields: 100
  on_exit: <function on_exit at 0x7fad4d3f49b0>
  config: None
  logconfig: None
  check_config: False
  do_handshake_on_connect: False
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x7fad4d3f4488>
  post_request: <function post_request at 0x7fad4d3f4578>
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x7fad4d3f4050>
  ca_certs: None
  threads: 1
  max_requests: 0
  chdir: /home/box/web
  daemon: False
  user: 1002
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  worker_exit: <function worker_exit at 0x7fad4d3f46e0>
  paste: None
  default_proc_name: hello:app
  errorlog: out
  logger_class: gunicorn.glogging.Logger
  syslog_addr: udp://localhost:514
  syslog_prefix: None
  ciphers: TLSv1
  worker_abort: <function worker_abort at 0x7fad4d3f41b8>
  loglevel: debug
  bind: ['192.168.101.128:8080']
  raw_env: []
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x7fad4d3f4848>
  timeout: 30
  on_starting: <function on_starting at 0x7fad4d46d758>
  debug: False
  django_settings: None
  tmp_upload_dir: None
  keyfile: None
  backlog: 2048
2016-04-01 14:16:13 [7334] [INFO] Starting gunicorn 19.0.0
2016-04-01 14:16:13 [7334] [DEBUG] Arbiter booted
2016-04-01 14:16:13 [7334] [INFO] Listening at: http://192.168.101.128:8080 (7334)
2016-04-01 14:16:13 [7334] [INFO] Using worker: sync
2016-04-01 14:16:13 [7339] [INFO] Booting worker with pid: 7339
2016-04-01 14:16:13 [7339] [ERROR] Exception in worker process:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gunicorn/arbiter.py", line 502, in spawn_worker
    worker.init_process()
  File "/usr/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python2.7/dist-packages/gunicorn/util.py", line 365, in import_app
    __import__(module)
  File "/home/box/web/hello.py", line 6
    '''   if environ['QUERY_STRING'] == '':
      body="empty"
   else:
      body="get is not null"
'''
                                           
                  
        
                            
  ^
IndentationError: expected an indented block
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gunicorn/arbiter.py", line 502, in spawn_worker
    worker.init_process()
  File "/usr/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python2.7/dist-packages/gunicorn/util.py", line 365, in import_app
    __import__(module)
  File "/home/box/web/hello.py", line 6
    '''   if environ['QUERY_STRING'] == '':
      body="empty"
   else:
      body="get is not null"
'''
                                           
                  
        
                            
  ^
IndentationError: expected an indented block
2016-04-01 14:16:13 [7339] [INFO] Worker exiting (pid: 7339)
2016-04-01 14:16:51 [7342] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  post_fork: <function post_fork at 0x7fda207b9cf8>
  pythonpath: None
  syslog: False
  syslog_facility: user
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 3
  cert_reqs: 0
  suppress_ragged_eofs: True
  when_ready: <function when_ready at 0x7fda207b9a28>
  pre_fork: <function pre_fork at 0x7fda207b9b90>
  preload_app: False
  workers: 1
  keepalive: 2
  accesslog: out
  pidfile: None
  worker_tmp_dir: None
  group: 1002
  graceful_timeout: 30
  spew: False
  proc_name: None
  umask: 0
  on_reload: <function on_reload at 0x7fda207b98c0>
  post_worker_init: <function post_worker_init at 0x7fda207b9e60>
  pre_exec: <function pre_exec at 0x7fda20740320>
  limit_request_fields: 100
  on_exit: <function on_exit at 0x7fda207409b0>
  config: None
  logconfig: None
  check_config: False
  do_handshake_on_connect: False
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x7fda20740488>
  post_request: <function post_request at 0x7fda20740578>
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x7fda20740050>
  ca_certs: None
  threads: 1
  max_requests: 0
  chdir: /home/box/web
  daemon: False
  user: 1002
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  worker_exit: <function worker_exit at 0x7fda207406e0>
  paste: None
  default_proc_name: hello:app
  errorlog: out
  logger_class: gunicorn.glogging.Logger
  syslog_addr: udp://localhost:514
  syslog_prefix: None
  ciphers: TLSv1
  worker_abort: <function worker_abort at 0x7fda207401b8>
  loglevel: debug
  bind: ['192.168.101.128:8080']
  raw_env: []
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x7fda20740848>
  timeout: 30
  on_starting: <function on_starting at 0x7fda207b9758>
  debug: False
  django_settings: None
  tmp_upload_dir: None
  keyfile: None
  backlog: 2048
2016-04-01 14:16:51 [7342] [INFO] Starting gunicorn 19.0.0
2016-04-01 14:16:51 [7342] [DEBUG] Arbiter booted
2016-04-01 14:16:51 [7342] [INFO] Listening at: http://192.168.101.128:8080 (7342)
2016-04-01 14:16:51 [7342] [INFO] Using worker: sync
2016-04-01 14:16:51 [7347] [INFO] Booting worker with pid: 7347
2016-04-01 14:16:59 [7347] [DEBUG] GET /
192.168.101.1 - - [01/Apr/2016:14:16:59 -0400] "GET / HTTP/1.1" 200 14 "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
2016-04-01 14:17:01 [7342] [INFO] Handling signal: int
2016-04-01 14:17:01 [7347] [INFO] Worker exiting (pid: 7347)
2016-04-01 14:17:01 [7342] [INFO] Shutting down: Master
2016-04-01 14:17:16 [7352] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  post_fork: <function post_fork at 0x7f7f5ef4acf8>
  pythonpath: None
  syslog: False
  syslog_facility: user
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 3
  cert_reqs: 0
  suppress_ragged_eofs: True
  when_ready: <function when_ready at 0x7f7f5ef4aa28>
  pre_fork: <function pre_fork at 0x7f7f5ef4ab90>
  preload_app: False
  workers: 1
  keepalive: 2
  accesslog: out
  pidfile: None
  worker_tmp_dir: None
  group: 1002
  graceful_timeout: 30
  spew: False
  proc_name: None
  umask: 0
  on_reload: <function on_reload at 0x7f7f5ef4a8c0>
  post_worker_init: <function post_worker_init at 0x7f7f5ef4ae60>
  pre_exec: <function pre_exec at 0x7f7f5eed1320>
  limit_request_fields: 100
  on_exit: <function on_exit at 0x7f7f5eed19b0>
  config: None
  logconfig: None
  check_config: False
  do_handshake_on_connect: False
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x7f7f5eed1488>
  post_request: <function post_request at 0x7f7f5eed1578>
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x7f7f5eed1050>
  ca_certs: None
  threads: 1
  max_requests: 0
  chdir: /home/box/web
  daemon: False
  user: 1002
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  worker_exit: <function worker_exit at 0x7f7f5eed16e0>
  paste: None
  default_proc_name: hello:app
  errorlog: out
  logger_class: gunicorn.glogging.Logger
  syslog_addr: udp://localhost:514
  syslog_prefix: None
  ciphers: TLSv1
  worker_abort: <function worker_abort at 0x7f7f5eed11b8>
  loglevel: debug
  bind: ['192.168.101.128:8080']
  raw_env: []
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x7f7f5eed1848>
  timeout: 30
  on_starting: <function on_starting at 0x7f7f5ef4a758>
  debug: False
  django_settings: None
  tmp_upload_dir: None
  keyfile: None
  backlog: 2048
2016-04-01 14:17:16 [7352] [INFO] Starting gunicorn 19.0.0
2016-04-01 14:17:16 [7352] [DEBUG] Arbiter booted
2016-04-01 14:17:16 [7352] [INFO] Listening at: http://192.168.101.128:8080 (7352)
2016-04-01 14:17:16 [7352] [INFO] Using worker: sync
2016-04-01 14:17:16 [7357] [INFO] Booting worker with pid: 7357
2016-04-01 14:17:18 [7357] [DEBUG] GET /
192.168.101.1 - - [01/Apr/2016:14:17:18 -0400] "GET / HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
2016-04-01 14:17:19 [7352] [INFO] Handling signal: int
2016-04-01 14:17:19 [7357] [INFO] Worker exiting (pid: 7357)
2016-04-01 14:17:20 [7352] [INFO] Shutting down: Master
